TAF_EasyRest/TestFramework/Pages/RestaurantMenuPageObject.cs

```diff
@@ -0,0 +1,34 @@
+﻿using TestFramework.PageComponents.NavigationMenuComponents;
+using TestFramework.PageComponents.RestaurantMenuComponents;
+
+namespace TestFramework.Pages
+{
+    public class RestaurantMenuPage : BasePage
```

Comment:
![image](https://user-images.githubusercontent.com/14205615/181525023-a86d791f-d63a-475c-b7bc-d5029ed2b3f8.png)
Rename in Solution Explorer.